earth_enterprise/rpms/opengee-server/post-uninstall.sh

```diff
@@ -21,20 +21,14 @@ set +x
 #-----------------------------------------------------------------
 main_postuninstall()
 {
-    # Fedora recommends to never delete users that a package has created:
-    #   https://fedoraproject.org/wiki/Packaging:UsersAndGroups
-    # TODO: Consider whether we want to remove this:
-    if getent passwd "$GEAPACHEUSER" >/dev/null; then
-        userdel "$GEAPACHEUSER"
-    fi
-    if getent passwd "$GEPGUSER" >/dev/null; then
-        userdel "$GEPGUSER"
-    fi
+    echo "opengee users $GEAPACHEUSER and $GEPGUSER may now safely be removed."
```

Comment:
Same comments as above.

Style: 'opengee' -> 'OpenGEE'

Minor: The message would be more accurate, if it said the users and groups can be removed, if there are no files or directories owned by them.

Reply:
true...let me do that now.